tests/core/test_graph.py

```diff
@@ -164,6 +164,20 @@ def test_graph_constructor_nodes_from_edges():
     assert g.node_types == {"abc"}
     assert sorted(g.nodes()) == [0, 1, 2, 4, 5]
 
+    with pytest.raises(TypeError, match="edges: expected dict, found <class 'int'>"):
```

Comment:
Oops, the formatting of these error messages are poor. I've opened https://github.com/stellargraph/stellargraph/issues/1259.